plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultQueryBuilder.java

```diff
@@ -107,6 +108,7 @@ public PreparedQuery prepareSelectQuery(
         }
 
         sql += getGroupBy(client, groupingSets);
+        setClientInfo(connection, "ApplicationName", "Trino-" + session.getQueryId());
```

Comment:
for something called "application name" i would expect the value to be a constant "Trino", or "Trino" with version appended